### PR TITLE
UVFS-6445: Add request Id for Druid calls

### PR DESCRIPTION
**Pull request by jpang-h4**
Summary
-
* These request IDs are set in controller context in
  main Tetration UI repo

Tests Implemented
-
rspec

Reviewers:
-
mikwat duaneatat

Subscribers:
-

### CODE REVIEW

**Review by duaneatat — APPROVED** [review]
looks good! 0.1.13 is compatible with main_dev ui so you can upgrade to 0.1.14 whenever.

File: lib/druid/client.rb [hunk 1]
```diff
@@ -12,7 +14,14 @@ def initialize(broker_url, opts = nil)
     def send(query)
       uri = broker_uri(query)
 
-      req = Net::HTTP::Post.new(uri.path, {'Content-Type' =>'application/json'})
+      headers = {}
+      headers[X_REQUEST_ID] = RequestStore.store[:h4_request_id] if
+        RequestStore.store[:h4_request_id]
+      headers[X_ILB_REQUEST_ID] = RequestStore.store[:ilb_request_id] if
```
Comment (mikwat) [inline lib/druid/client.rb:22]: The https://github.com/steveklabnik/request_store gem should be added https://github.com/TetrationAnalytics/ruby-druid/blob/master/Gemfile#L8

File: spec/lib/client_spec.rb [hunk 2]
```diff
@@ -1,4 +1,13 @@
 describe Druid::Client do
+  let(:h4_request_id) { 'lb-1234' }
+  let(:ilb_request_id) { 'ilb-1234' }
+
+  before do
+    RequestStore = class_double("RequestStore").
+      as_stubbed_const(:transfer_nested_constants => true)
```
Comment (mikwat) [inline spec/lib/client_spec.rb]: Once you include the gem, you shouldn't need to create a `class_double`.

**Review by mikwat — APPROVED** [review]

File: lib/druid/client.rb (hunk 1 above)
Comment (jpang-h4) [reply lib/druid/client.rb:22]: sure 👍

**Pull request merged**